README.markdown

```diff
@@ -1,22 +1,21 @@
 # Home Assistant website
 
-This is the source for the [Home-Assistant.io website](https://home-assistant.io) for the [Home Assistant project](https://github.com/home-assistant/home-assistant)
+The source of the [Home-Assistant.io website](https://home-assistant.io), for the [Home Assistant project](https://github.com/home-assistant/home-assistant)
```

Comment:
Let's stick with the start `This is the source for` but remove the `for the Home Assistant project` part.